docs/_pages/docs/03-understand/04-bonds.md

```diff
@@ -2,10 +2,54 @@
+
+By default, the bond is 3% of the total trade amount. Alternatively, order makers can customize this amount to be anywhere from 2% to 15%. Larger bonds mean more "skin in the game" that is required to trade.
+
+The bond does not leave your Lightning wallet, but please know some wallets play nicer with RoboSats than others due to the nature of the Lightning hold invoice mechanic. Refer to [Understand > Wallets](https://learn.robosats.com/docs/wallets/) for additional information.
+
+*Note: The option allowing "Bondless Takers" is in the works but not available at the moment.*
+
+## **How to Lock a Bond**
+
+First, refer to [Understand > Wallets](https://learn.robosats.com/docs/wallets/) for compatible Lightning wallets that will help make using RoboSats a smoother experience. Depending on the wallet, the invoice might show as a payment that is in transit, frozen, or even appearing to fail. Check the wallet compatability list!
+
+Read the relevant guide depending on if you are making or taking the order:
+* **Maker**: Select "Make Order" and modify the order conditions to your liking. The order can be customized to require a fidelity bond other than the default 3% of the total trade amount, ranging anywhere from 2% to 15%. Once complete, confirm with "Create Order" and then use the following QR code found in the "Contract Box" with your Lightning wallet to lock the indicated amount of sats for your fidelity bond. *Note: Be prepared with your wallet beforehand because the order box expires in ten minutes.*
+* **Taker**: Browse the order book and find an order to your liking. Simply select the "Take Order" option and then use the following QR code found in the "Contract Box" with your Lightning wallet to lock the indicated amount of sats for your fidelity bond. *Note: Be prepared with your wallet beforehand because the order box expires in four minutes. If you do not proceed, the taken order is made public again.*
+
+After the trade is completed and both robots are satisfied, the maker and taker bonds are relinquished. Technically, the locked bond never left your wallet; but take caution, if you fail to follow the contract obligations by trying to cheat or cancelling unilaterally, you will forfeit your fidelity bond.
+
+Your wallet may take a while for funds to show as relinquished on your account balance. Some wallets have difficulty with recognizing the Lightning hold invoice as a temporary hold on your funds.
+
+If the issue persists, please reach out to the RoboSats Telegram group; but beware of scammers that may directly contact you and impersonate RoboSats staff! RoboSats staff will never directly contact you first. See [Contribute > Code > Communication Channels](https://learn.robosats.com/contribute/code/#communication-channels) for available Telegram groups. 
+
+## **Losing Your Bond**
```

Comment:
We could add some bullet point summarizing the conditions in which you lose the bond (basically these 4):
- Trying to cheat or deceive your peer (i.e., losing a dispute)
- As a buyer: failing to submit a payout invoice in time.
- As a seller: failing to post the trade collateral in time.
- As a seller: failing to release the escrow Satoshis to the buyer by confirming "fiat received".

Reply:
Agreed, I like bullet points

Would a fifth condition for losing the bond be a peer trying to unilaterally cancel the order? Or is this perceived the same as trying to cheat or deceive peer

Reply:
I've gone ahead and thrown in the 5th bullet point just to make clear that cancelling uncollaboratively will result in lost bond